curriculum/challenges/english/02-javascript-algorithms-and-data-structures/functional-programming/use-higher-order-functions-map-filter-or-reduce-to-solve-a-complex-problem.english.md

```diff
@@ -0,0 +1,84 @@
+---
+id: 587d7b88367417b2b2512b45
+title: Use Higher-Order Functions map, filter, or reduce to Solve a Complex Problem 
+challengeType: 1
+---
+
+## Description
+<section id='description'>
+Now that you have worked through a few challenges using higher-order functions like <code>map()</code>, <code>filter()</code>, and <code>reduce()</code>, you get to work on a slightly more complex problem allowing you to choose which higher-order function(s) best solves it.
```

Comment:
```suggestion
Now that you have worked through a few challenges using higher-order functions like <code>map()</code>, <code>filter()</code>, and <code>reduce()</code>, you get to work on a slightly more complex problem allowing you to choose which higher-order function(s) best solve(s) it.
```

Reply:
Maybe I can reword this to avoid having to use `function(s)` and `solve(s)`.  I will think come up with something else.

Reply:
@thecodingaviator What do you think about the following sentence?
> Now that you have worked through a few challenges using higher-order functions like <code>map()</code>, <code>filter()</code>, and <code>reduce()</code>, you will now apply them on a slightly more complex problem.

Reply:
I've made the change to the suggestion with a few changes